### DISCUSSION TITLE
where is osu! stable source code like lazer?

### DISCUSSION THREAD

**Discussion by NamOpt** [ppy/osu#26097] [Q&A] [answered]
i like developing osu! stable not lazer

**Comment by peppy** [comment, marked as answer]
It's not open source.

**Reply by fraljer** [reply]
peppy just release lazer transitional source code man please I'm begging you I know you have it in the osustable repo please man please please please please?

**Reply by rorogu922** [reply]
> peppy just release lazer transitional source code man please I'm begging you I know you have it in the osustable repo please man please please please please?

bruh
